src/Bicep.Cli/Helpers/ArgumentHelper.cs

```diff
@@ -44,6 +44,10 @@ public static void ValidateBicepParamFile(Uri fileUri)
         {
             throw new CommandLineException(string.Format(CliResources.UnrecognizedBicepparamsFileExtensionMessage, fileUri.LocalPath));
         }
+        if (!File.Exists(fileUri.LocalPath))
```

Comment:
@shenglol could you recommend the right interface to use for this?

Reply:
`IFileExplorer` and `IFileHandle` replace `IFileResolver`, and `IOUri` replaces `Uri`. It's fine to continue using `System.File` or `IFileSystem` for file existence checks for now. I'll update the entire `Bicep.Cli` project to use the new file I/O types later.

Reply:
thanks 👍